src/app/common/KumaLogo.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+import { useEnv } from '@/utilities'
+const env = useEnv()
+</script>
+<template>
+  <img
+    src="@/assets/images/product-logo.png"
+    :alt="`${env('KUMA_PRODUCT_NAME')} Logo`"
+  >
+</template>
+<style lang="scss" scoped>
+img {
+  max-height: 36px;
+}
+</style>
```

Comment:
**Suggestion**: Can we please stick to one style of arranging single file components and use some empty new lines to separate blocks of things?

```suggestion
<template>
  <img
    src="@/assets/images/product-logo.png"
    :alt="`${env('KUMA_PRODUCT_NAME')} Logo`"
  >
</template>

<script lang="ts" setup>
import { useEnv } from '@/utilities'

const env = useEnv()
</script>

<style lang="scss" scoped>
img {
  max-height: 36px;
}
</style>
```

Reply:
Yeah sure, added a couple of blank lines